### ISSUE TITLE
Do not include the new "Card" content type in the default OOTB search settings

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5751]
This is something that site admins/builders should be able to change, but as it is our definition of a card is:

> A Card can only be seen as part of another page. ...

So I don't think that it makes much sense to be providing is as an option in the "Only of the type(s)" filter of the advanced search:

![image](https://user-images.githubusercontent.com/2423362/187295227-b251c0a0-f538-4451-807a-8e838b0a72c7.png)

We should remove it from the default OOTB "Enabled content types" configuration in `/admin/config/search/settings`.

**Comment by olafgrabienski** [comment]
> We should remove [pageless content types] from the default OOTB "Enabled content types" configuration ...

Good point! I'm not sure if pageless content types should be *removed* from the "Enabled content types" in the search settings. At first sight, I'd prefer to *disable* them.

**Comment by kiamlaluno** [comment]
Considering that search results would give a link to nodes matching the search criteria, and those pageless items won't be accessible at their URLs, it doesn't make sense to include them for users who don't have the permission to access them from their URLs.
A user who can access pageless content using a URL could probably prefer to find them too.

The search results should already remove content items for which the logged-in user doesn't have the permission to see.
